barretenberg/cpp/src/barretenberg/flavor/goblin_ultra.hpp

```diff
@@ -287,7 +287,20 @@ class GoblinUltraFlavor {
      * circuits.
      * @todo TODO(https://github.com/AztecProtocol/barretenberg/issues/876)
      */
-    using VerificationKey = VerificationKey_<PrecomputedEntities<Commitment>, VerifierCommitmentKey>;
+    class VerificationKey : public VerificationKey_<PrecomputedEntities<Commitment>, VerifierCommitmentKey> {
```

Comment:
Needed a derived class to add the public_inputs in as public_inputs depends on the flavor::FF.